apps/token-manager/contracts/TokenManager.sol

```diff
@@ -320,7 +320,12 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
 
     function _mint(address _receiver, uint256 _amount) internal {
         require(_isBalanceIncreaseAllowed(_receiver, _amount), ERROR_BALANCE_INCREASE_NOT_ALLOWED);
+
+        // We check that, after minting, the total supply increases.
+        // Old MiniMeToken's overflow if totalSupply > 2 ** 128 - 1.
+        uint256 supplyBeforeMint = token.totalSupply();
```

Comment:
Let's use [`staticTotalSupply()`](https://github.com/aragon/aragonOS/blob/next/contracts/common/SafeERC20.sol#L161) to do this and the next check